R/nested_closeable_modal.R

```diff
@@ -1,6 +1,6 @@
 #' Nested Closeable Modal Popup
 #'
-#' @description `r lifecycle::badge("experimental")`
```

Comment:
I think this guy can be deprecated. It is not used anymore in our framework. WDYT @vedhav ?